Mathlib/Algebra/Algebra/Equiv.lean

```diff
@@ -754,3 +754,20 @@ def toAlgAut : G →* A ≃ₐ[R] A where
 end
 
 end MulSemiringAction
+
+section
+
+variable {R S T : Type*} [CommSemiring R] [Semiring S] [Semiring T] [Algebra R S] [Algebra R T]
+
+instance [Subsingleton S] [Subsingleton T] : Unique (S ≃ₐ[R] T) where
```

Comment:
Note that `instance [Subsingleton S] [Subsingleton T] : Subsingleton (S ≃ₐ[R] T) := inferInstance` already succeeds here; so maybe we should promote the existing `Subsingleton` instance to `Unique` rather than making a new one.